content/lessons/2016/inscribed-rectangle-problem/index.md

```diff
@@ -1,10 +1,320 @@
+
+We can place one of the two values on a new y-axis, using a second interval, so that each pair of values between $0$ and $1$ (and thus each pair of points on the loop) is associated with a single point in a $1 \times 1$ square.
+
+{{< figure image="pair-to-square.png" >}}
+By giving each of the two numbers (corresponding to our original two points on the loop) a unique axis to live on, we can represent any pair of points on the loop with a single point in a $1 \times 1$ square of 2D space.
+{{< /figure >}}
+
+Remember, we’re trying to find a surface that naturally represents the set of all pairs of points on the loop. This square is the first step to doing that.
+
+**Stop to think:** If you have a pair of points on the loop, how do you find the corresponding point on the square?
+
+**Answer:** {{< spoiler >}}Convert each loop point to a number between 0 and 1, and then use those two numbers as the x and y coordinates in the square.{{< /spoiler >}}
+
+The problem is that there’s some ambiguity when it comes to the edges of this square. Remember, the endpoints $0$ and $1$ on the interval really correspond to the same point of the loop, so they need to be glued together if we are to faithfully map back to the loop.
+
+So all the points on the left edge of the square really represent the same pair of points on the loop as the corresponding coordinates on the right edge of the square. If this square is going to represent pairs on the loop in a unique way, we’ll need to glue this left edge to the right edge.
+
+Likewise, the bottom edge needs to be glued to the top edge, since the y-coordinates of $0$ and $1$ each really represent the same second point in a given pair of points on the loop.
+
+Let’s mark each edge with some arrows to remember how the edges should be lined up:
+
+{{< figure image="square-alignment-arrows.png" >}}
+Matching points along opposite edges of the square both correspond to the same pair of points on the loop. To represent this faithfully, we need to glue together the edges of the square as indicated by the arrows.
+{{< /figure >}}
+
+If you bend this square to perform the gluing, first rolling it into a cylinder to glue the left and right edges, then gluing the ends of the cylinder, which represent the top and bottom edges, we get a torus (better known as the surface of a doughnut).
+
+{{< figure image="roll-into-torus.png" video="roll-into-torus.mp4" >}}
+When the square is wrapped up and its edges glued together following the arrows, it becomes a torus.
+{{< /figure >}}
+
+By gluing together the corresponding edges, we’ve solved the problem of duplicate mappings. Every individual point on this torus corresponds to a *unique* pair of points on the closed loop, and likewise every pair of points on the loop corresponds to a unique point on the torus.
+
+{{< figure image="torus-and-loop-correspondence.png" >}}
+There is a unique correspondence between pairs of points on the loop and individual points on the torus.
```

Comment:
Combine this image with the one below it, making this image the top of the next one.